tests/test-project/src/test/workbench/notifications.test.ts

```diff
@@ -134,13 +132,31 @@ describe('NotificationsCenter', () => {
 		it('get error notification works', async () => {
 			await new Workbench().executeCommand('Error Message');
 			await center.getDriver().sleep(200);
-			center = await new Workbench().openNotificationsCenter();
-			notification = (await center.getNotifications(NotificationType.Error))[0];
-
+			await waitForNotification(NotificationType.Error);
 			expect(await notification.getMessage()).to.equal('This is an error!');
 			expect(await notification.getType()).to.equal(NotificationType.Error);
 			await notification.dismiss();
 			await center.getDriver().wait(until.stalenessOf(notification));
 		});
+
+		async function waitForNotification(NotificationType: NotificationType) {
```

Comment:
I am wondering.. why the function argument is using `PascalCase` instead of `camelCase`?